docs_2.0/manual_EN/5.deployment-and-installation/2.compile-and-install-nebula-graph/1.install-nebula-graph-by-compiling-the-source-code.md

```diff
@@ -0,0 +1,122 @@
+# Install Nebula Graph by compiling the source code
+
+Installing Nebula Graph from the source code allows you to customize the compiling and installation settings and test the latest features.
+
+## Prerequisites
+
+* You have prepared the necessary resources described in [Prepare resources for compiling, installing, and running Nebula Graph](../1.resource-preparations.md/).
+
+* You can access the Internet from the host you plan to install Nebula Graph.
+
+## How to install
+
+1. Use Git to clone the source code of Nebula Graph to your host.
+
+    ```bash
+    $ git clone https://github.com/vesoft-inc/nebula-graph.git
+    ```
+
+2. Make the `nebula-graph` directory the current working directory.
+
+    ```bash
+    & cd nebula-graph
+    ```
+
+3. Create a `build` directory and make it the current working directory.
+
+    ```bash
+    & mkdir build && cd build
+    ```
+
+4. Generate the Makefile with CMake.
+
+    > **NOTE**:
+    >
+    >* The installation path is /user/local/nebula by default. To customize it, add the `-DCMAKE_INSTALL_PREFIX=/your/install/path/` CMake variable in the following command.
+    >* For the instructions of more CMake variables, see [CMake variables](#cmake-variables).
+
+    ```bash
+    $ cmake -DENABLE_BUILD_STORAGE=on -DENABLE_TESTING=OFF -DCMAKE_BUILD_TYPE=Release ..
+    ```
+
+5. Compile Nebula Graph.
+
+    Assuming *cores* is the number of CPU cores and *mem_gb* is the memory size (in GB), to appropriately speed up the compiling, you can use the value of the smaller one between cores and mem_gb/2 as the value of N in the following command.
+
+    ```bash
+    $ make -jN
+    ```
+
+6. Install Nebula Graph.
+
+    ```bash
+    $ make install-all
+    ```
+
+## CMake variables
+
+Usage of CMake variables:
+
+```bash
+$ cmake -DVariable=<value> ...
+```
+
+The following CMake variables can be used at the configure (cmake) stage to adjust the compiling settings.
+
+### ENABLE_BUILD_STORAGE
+
+The Nebula Graph 2.0 series support separated compute and storage. The `ENABLE_BUILD_STORAGE` variable is set to `OFF` by default so that the storage service is not installed together with the graph service.
```

Comment:
what are `series`?

Reply:
Should be 2.x series. It means all versions starting with 2. .

Reply:
> Should be 2.x series. It means all versions starting with 2. .

it's OK to say version 2.x (no series)